### PR TITLE
Added delete_many support to start supporting bulk deletes on OL

### PR DESCRIPTION
**Pull request by BharatKalluri**

### CODE REVIEW

File: olclient/openlibrary.py [hunk 1]
```diff
@@ -131,6 +131,19 @@ def save_many(self, docs, comment):
...
+            'Opt': '"http://openlibrary.org/dev/docs/api"; ns=42',
+            '42-comment': comment
+        }
+        payload = [{
+            'key': f'/{self.get_type(ol_id)}s/{ol_id}',
+            'type': {
+                'key': '/type/delete'
+            }
+        } for ol_id in ol_ids]
+        return self.session.post(f'{self.base_url}/api/save_many', json.dumps(payload), headers=headers)
```
Comment (cdrini) [inline olclient/openlibrary.py:145]: Oh, might be nice to call self.save_many, to keep this DRY. Also we generally call these `olids` (no underscore)

```suggestion
    def delete_many(self, olids: List[str], comment: str):
        docs = [{
            'key': f'/{self.get_type(olid)}s/{olid}',
            'type': {
                'key': '/type/delete'
            }
        } for olid in olids]
        return self.save_many(docs, comment)
```

Comment (cdrini) [reply olclient/openlibrary.py:145]: Scratch that, it looks like save_many won't respond to a dict; it needs  document with a `.json()` method. So we should be able to do something like this (but not sure whether `Delete` accessible at this point, or needs some prefix).

```suggestion
    def delete_many(self, olids: List[str], comment: str):
        return self.save_many([Delete(olid) for olid in olids], comment)
```

Comment (BharatKalluri) [reply olclient/openlibrary.py]: using `Delete` now, thanks for the suggestion

**Review by cdrini — APPROVED** [review]
Lgtm! We tested in @BharatKalluri colab with ` ol.delete_many(['OL10630219M', 'OL7985836W'], 'dumpbin cleanup')` , and it correctly deleted the records: https://openlibrary.org/recentchanges/2021/06/08/bulk_update/83203107

**Pull request merged**